RtspServer/Classes/SourceMedia.cs

```diff
@@ -239,15 +238,15 @@ public SourceMedia(string name, Uri source, NetworkCredential sourceCredential,
 
         internal void OnFrameDecoded(System.Drawing.Image decoded)
         {
-            if (DecodeFrames && decoded != null && FrameDecoded != null)
+            if (DecodeFrames && decoded != null)
```

Comment:
Lets either add the check back here or use ?.Invoke, it's up to you which way we go.

Reply:
I made the edit just now, thank you!